sys/include/sema.h

```diff
@@ -33,23 +35,35 @@ extern "C" {
 /**
  * @brief   Creates semaphore statically.
  *
- * @param[in] value Initial value for the semaphore.
+ * @param[in] value Initial value for the semaphore (can't be 0). For a 0
+ *                  initialized semaphore @see SEMA_CREATE_EMPTY
```

Comment:
1. `SEMA_CREATE_EMPTY` does not exist
2. I don't understand the reasoning behind this restriction.